alerts/state.libsonnet

```diff
@@ -0,0 +1,37 @@
+{
+  prometheusAlerts+:: {
```

Comment:
This can be a bit confusing. You define alerting rules, here. The alerts is what prometheus sends/stores when any of the rules is hit.

It can get pretty confusing when you call the alerting rules simply alerts (I know it was for me at first). Can we signify somehow that these are alerting rules?

Reply:
@b-ranto I am not sure this keyword could be changed. Under this dictionary kind of thing, ideally all the alerting rules needs to be defined.

Reply:
@b-ranto @shtripat This can be changed if required. But I wouldn't recommend it as this is the standard that's being followed by kubernetes-mixins (https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/19db38fc449df024446059f21d5a329babaa3927/lib/alerts.jsonnet#L1)